pkgs/applications/misc/kiwix/default.nix

```diff
@@ -105,5 +105,6 @@ stdenv.mkDerivation rec {
     knownVulnerabilities = [
       "CVE-2015-1032"
     ];
+    broken = true; # marked broken at 2020-03-26
```

Comment:
Why is it marked broken?

Reply:
Because it's broken right now. Fix in #117851